dataprofiler/profilers/profile_builder.py

```diff
@@ -2801,6 +2829,11 @@ def _update_profile_from_chunk(
         elif isinstance(data, list):
             data = pd.DataFrame(data, dtype=object)
 
+        # If options.multiprocess is enabled, auto-toggle multiprocessing
+        auto_multiprocess_toggle = None
+        if self.options.multiprocess.is_enabled:
+            auto_multiprocess_toggle = self._auto_multiprocess_toggle(data, 750000, 20)
```

Comment:
since the default is set, no need to pass in as well.

Reply:
+1

Reply:
+1

Reply:
fixed